### ISSUE TITLE
[1.7.10] Where is the 1.7.10 Source code?

### ISSUE THREAD

**Issue by ghost** [MatrexsVigil/harvestcraft#370]
I want to backup all mod's source code used by my server., because a lot of 1.7.10 mod abandoned.
But I can't find Pam's Harverstcraft 1.7.10 code. Where is it? Thank you very much!

**Comment by SourcePowered** [comment]
#14 Duplicate.

**Comment by ghost** [comment]
Duplicate of #14

**Comment by Septem151** [comment]
Still waiting on deobf source code...

**Comment by CorporalData** [comment]
same :P

really, the only thing I'm looking for is fixing a single issue, that being the missing texture for the chocolate roll. I have no idea why it's missing one, since the texture is clearly in the mod files. any help would be appreciated.
